docker-compose.yml

```diff
@@ -5,11 +5,11 @@ server:
   volumes:
    - .:/src
    - /var/run/docker.sock:/tmp/docker.sock
-   - ./config.yml:/config.yml
+   - ~/volumes/ci/roger:/roger/
```

Comment:
we probably shouldn't do it -- the `docker-compose.yml` is a file used for development only and we should give a development environment that kinda works out of the box, without you having to create stuff manually